src/containers/Tenant/Diagnostics/Diagnostics.tsx

```diff
@@ -83,7 +83,11 @@ function Diagnostics(props: DiagnosticsProps) {
             return diagnosticsTab;
         } else {
             const newPage = pages[0].id;
-            forwardToDiagnosticTab(newPage);
+
+            if (wasLoaded) {
+                forwardToDiagnosticTab(newPage);
+            }
+
             return newPage;
```

Comment:
this kinda breaks the logic flow
wouldn't it make more sense to check this at the start of this function? And return, say, `undefined` in case `wasLoaded === false`